exchanges/okx/helpers.go

```diff
@@ -0,0 +1,214 @@
+package okx
+
+import (
+	"fmt"
+	"slices"
+	"strings"
+
+	"github.com/thrasher-corp/gocryptotrader/common"
+	"github.com/thrasher-corp/gocryptotrader/currency"
+	"github.com/thrasher-corp/gocryptotrader/exchanges/asset"
+	"github.com/thrasher-corp/gocryptotrader/exchanges/order"
+)
+
+// OrderTypeFromString returns order.Type instance from string
+func OrderTypeFromString(orderType string) (order.Type, error) {
```

Comment:
Nit: Think these should be private funcs 🙂 
Sorry I didn't think about that before.

Reply:
I prefer it as a private function, However, Someone, maybe you, did recommend me to capitalize these functions and I did so ...

Reply:
Definitely wasn't me. I prefer private functions because it reduces our risk footprint with users; We know all paths that will call a function.

I can't think of why we'd need these to be public.

Reply:
That was me; I bypass IBotExchange a lot and hook directly into the exchange.go functions, their params sometimes require formatting or some other prevision which is done by these helpers, I also have on occasion written tests that confirm functions as well externally. I prefer everything to be exported for an exchange library. That being said it is more concise and I can just expose what I need on the branch I am working on anyway. So nothing here that will stop a merge.

> reduces our risk footprint with users
@gbjk What risk foot print?

Reply:
I was thinking about your use-cases when I made this comment.
I couldn't think of why you'd want to call these specific functions externally, but if you have a use case then I'll yield to that.

> What risk foot print?

Size of exported code at risk of bugs from unexpected inputs or usages.